server/utils/characteristicUtils.ts

```diff
@@ -0,0 +1,11 @@
+import { Characteristic } from '../@types/shared'
+
+export const prepareCharacteristics = (
```

Comment:
'prepare' is a bit vague as a name to me but I can't suggest anything better and the function body is very clear so I think it's fine for now.

Reply:
Yeah, I don't like it but best I could come up with was that or filterAndSortCharacteristics

Reply:
Actually, typing that out, maybe that's fine, brb 😛